lib/Core/Algorithm/src/UpdateFunctions.c

```diff
@@ -619,7 +637,13 @@ void _GenerateObservationCovariance_AHRS(void)
      * ----- Yaw -----
      * CHANGED TO SWITCH BETWEEN GPS AND MAG UPDATES
      */
-    if ( useGpsHeading )
+    if (useRTKHeading) {
+        // from a numerical point of view I should do like that
+        // or sigma_deg^2 * def2rad^2 ?
```

Comment:
wel ... yes ... you do need to convert to radians.

But then it would probably best to convert to radians in the Ublox parsing code directly, no ?